pkg/errors/errors.go

```diff
@@ -50,6 +50,10 @@ func (e Error) Error() string {
 // forms a more readable stack trace.
 type Op string
 
+func (o Op) String() string {
+	return string(o)
```

Comment:
@manugupt1 I don't recall 100% but this might be an infinite recursion. I could be wrong but would you mind writing a quick test for it? I would say in a follow-up PR but I think it's used in critical path code